src/parity_testing/auth/testSuites/confirmSignIn/Test_that_SignIn_with_SMS_challenge_invokes_proper_cognito_request_and_returns_success.json

```diff
@@ -9,11 +9,11 @@
                 "apiName": "respondToAuthChallenge",
                 "responseType": "success",
                 "response": {
-                    "authenticationResult": {
-                        "idToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1c2VySWQiLCJ1c2VybmFtZSI6InVzZXJuYW1lIiwiZXhwIjoxNTE2MjM5MDIyLCJvcmlnaW5fanRpIjoib3JpZ2luX2p0aSJ9.Xqa-vjJe5wwwsqeRAdHf8kTBn_rYSkDn2lB7xj9Z1xU",
-                        "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1c2VySWQiLCJ1c2VybmFtZSI6InVzZXJuYW1lIiwiZXhwIjoxNTE2MjM5MDIyLCJvcmlnaW5fanRpIjoib3JpZ2luX2p0aSJ9.Xqa-vjJe5wwwsqeRAdHf8kTBn_rYSkDn2lB7xj9Z1xU",
-                        "refreshToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1c2VySWQiLCJ1c2VybmFtZSI6InVzZXJuYW1lIiwiZXhwIjoxNTE2MjM5MDIyLCJvcmlnaW5fanRpIjoib3JpZ2luX2p0aSJ9.Xqa-vjJe5wwwsqeRAdHf8kTBn_rYSkDn2lB7xj9Z1xU",
-                        "expiresIn": 300
+                    "AuthenticationResult": {
```

Comment:
Using `PascalCase` instead of `camelCase` to match with how Cognito sends the response. This makes easy deserialization by directly utilizing sdk utility methods.

Reply:
Is there a reason you want to replicate case in Cognito APIs response than Kotlin SDKs CIPs response object?

Reply:
I want to replicate that behaviour because we can resuse serialization/deserialization code from the SDK